src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -215,6 +215,10 @@ static void checkArg(string arg, string name)
                 }
                 TraceEventSession.Merge(files.ToArray(), traceFileName);
 
+                if (guiApp)
+                {
+                    appExe = Path.Join(workingDir, appExe);
```

Comment:
wondering why we need to join the paths here; seems evt.commandLine only takes whatever appExe is

Reply:
in the GUI app case the event payload includes the full path to the exe. We happen to know (by convention) that the specified working dir is always the root to the app to run. We could change this to always pass in the full path also if this becomes problematic.

Reply:
"We happen to know (by convention) that the specified working dir is always the root to the app to run." not true for sdk tests but it's fine as long as guiApp=false for sdk tests...

Reply:
fair point. though maybe we should rethink that. if SDK tests are so different we'll have trouble sharing templates etc..